CRM/Upgrade/Incremental/php/FiveSixtyFour.php

```diff
@@ -33,6 +35,7 @@ public function upgrade_5_64_alpha1($rev): void {
     $this->addTask('Drop unused civicrm_action_mapping table', 'dropTable', 'civicrm_action_mapping');
     $this->addTask('Update post_URL/cancel_URL in logging tables', 'updateLogging');
     $this->addTask('Add in Everybody ACL Role option value', 'addEveryBodyAclOptionValue');
+    $this->addTask(ts('Fix double json encoding of accepted_credit_cards field in payment processor table'), 'fixDoubleEscapingPaymentProcessorCreditCards');
```

Comment:
`ts` is unusual here, since it's just a one-off message for admins.

Reply:
There is some ts usage here https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/php/FiveSixtyThree.php#L40 so *shrugs*

Reply:
Those are common strings repeated across upgrades, so are better candidates for ts.

Reply:
Although I'd argue they are still just for admins so shouldn't have ts either.

Reply:
I've removed the ts now